src/components/CityWeatherShortcut/CityWeatherShortcut.tsx

```diff
@@ -44,7 +44,7 @@ const CityWeatherShortcut = ({ cityName, iconType }: Props) => {
           </Temperature>
           <City>{name}</City>
           <ButtonWrapper>
-            <FavoritesButton cityName={cityName} size={20} iconType={iconType} />
+            <FavoritesButton cityName={name + ',' + country} size={20} iconType={iconType} />
```

Comment:
**sugegstion** enum IconTypes shoul not be named IconTypes, just Icon. 

In my opinion, type refers to types such as outlined, small, medium etc.

Reply:
Okay, done :)